lib/scrape.js

```diff
@@ -53,12 +54,6 @@ const CHAT_IMAGES = {
   OTHER: 'images/chat.png',
 }
 
-const GH_API_OPTIONS = {
-  headers: process.env.GITHUB_TOKEN
-    ? { Authorization: `token ${process.env.GITHUB_TOKEN}` }
```

Comment:
keep this, and keep GH_API_BASE, as I am sure we will want to access some features are not available via Graphql, and we dont want people to need to re-invent this, and possibly pick a new token variable name, etc.

Add a note that `GH_API_OPTIONS` is unused.